solidity/contracts/liquidity-protection/LiquidityProtectionStats.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+import "./interfaces/ILiquidityProtectionStats.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
```

Comment:
Ideally, contracts from external repos should come first, thus helping us detect issues when we're trying to redefine (or in conflict with) external changes

Reply:
I usually go by the order of inheritance in the contract declaration (where I always place the contract's own interface first), but OK.